marketplace/deployer_helm_tiller_base/test/post-delete.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
```

Comment:
Creation of these are more acceptable, as these objects are created not by the deployer but by the user (or server side tiller).

There is still a concern that we rely on `cluster-admin` role. The user might not have this permission (note that not only role creation but binding operations are also checked; somehow it works with IAM integration).

Since this only triggers when the user enters the CLI world, I think it's acceptable for now.

Probably the better way is to keep the deployer's service account around to run all hooks. (See my comment about alternate approach to owner referencing using container lifecycle).

Reply:
Let's discuss. :)